Libplanet.Analyzers/rules/LAA1002.md

```diff
@@ -0,0 +1,53 @@
+<def>LAA1002</dev>: DictionariesOrSetsShouldBeOrderedToEnumerate
```

Comment:
When I compared this line with the [LAA1001 document](https://github.com/planetarium/libplanet/blob/a75e590c15ecfc610b86a8f4047aaa4abc36194d/Libplanet.Analyzers/rules/LAA1001.md), it seems a typo. Doesn't it?

> `<def>LAA1001</def>: SystemRandomBreaksActionDeterminism`
```suggestion
<def>LAA1002</def>: DictionariesOrSetsShouldBeOrderedToEnumerate
```

Reply:
Fixed!